react-delegationdashboard/src/components/Overview/Cards/index.tsx

```diff
@@ -175,7 +180,7 @@ const Views = () => {
         )
       )}
 
-      {isAdmin() && location.pathname === '/owner' && (
+      {isAdmin() && isOwner() && (
```

Comment:
The naming of this functions became pretty confusing. Owner would have been a better naming from the start, could you please rename `isAdmin` to `isOwner` and `isOwner` to `isOwnerPath` or something like that?

Reply:
Right. Makes sense.